Modules/_sqlite/connection.c

```diff
@@ -645,14 +668,8 @@ _pysqlite_func_callback(sqlite3_context *context, int argc, sqlite3_value **argv
         Py_DECREF(py_retval);
     }
     if (!ok) {
-        pysqlite_state *state = pysqlite_get_state(NULL);
-        if (state->enable_callback_tracebacks) {
-            PyErr_Print();
-        }
-        else {
-            PyErr_Clear();
-        }
-        sqlite3_result_error(context, "user-defined function raised exception", -1);
+        set_sqlite_error(context,
+                "user-defined function raised exception");
```

Comment:
After renaming, this is now within the 79 char limit :)

```suggestion
        set_sqlite_error(context, "user-defined function raised exception");
```